changelog/3605.bugfix.rst

```diff
@@ -0,0 +1 @@
+unpack marks from parameterization to prevent the markdecorator missmatch bug.
```

Comment:
How can we phrase this in a way that end users (that don't know the internals) can understand? 🤔